site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -127,7 +127,18 @@ public function addPerson($queue_code) {
                 );
             }
             else {
-                $contact_info = $_POST['contact_info'];
+                //make sure contact information matches instructors regex pattern
+                $regex_pattern = $this->core->getQueries()->getQueueRegex($queue_code)[0]['regex_pattern'];
+                $regex_pattern = '#' . $regex_pattern . '#';
+                if ($regex_pattern !== '') {
```

Comment:
If that queue has no regex pattern configured, then on line 131 `$regex_pattern` is equal to `''`. If on the next line you concatenate the `#` symbol to the beginning and the end of the string, should the if statement not check for `$regex_pattern !== '##'` ? (Or move the concatenation of the pound sign into the if statement) Maybe I'm misunderstanding

Reply:
thanks for catching that. I just moved the concatenation